tests/el/_fixtures.yaml

```diff
@@ -1,20 +1,20 @@
 language: el
 areas:
-  - name: Kitchen
+  - name: Κουζίνα
     id: kitchen
-  - name: Living Room
+  - name: Σαλόνι
     id: living_room
-  - name: Bedroom
+  - name: Υπνοδωμάτιο
     id: bedroom
-  - name: Garage
+  - name: Γκαράζ
     id: garage
 entities:
-  - name: Bedroom Lamp
+  - name: Φως υπνοδωματίου
     id: light.bedroom_lamp
     area: bedroom
-  - name: Kitchen Switch
+  - name: Φως κουζίνας
     id: switch.kitchen
     area: kitchen
-  - name: Ceiling Fan
+  - name: Ανεμιστήρας οροφής
```

Comment:
```suggestion
  - name: Ανεμιστήρας Οροφής
```

Reply:
-There is no need for change here, as the text for training the model is not case-sensitive. Moreover, the correct greek grammar format is the originally suggested one.